metis-core/metis-core-rest/src/main/resources/application.properties.example

```diff
@@ -100,3 +102,21 @@ metis.core.baseUrl=
 metis.core.max.served.execution.list.length=
 metis.core.max.depublish.record.ids.per.dataset=
 metis.link.checking.default.sampling.size=
+
+#Actuator
```

Comment:
my suggestion is add that this for kubernetes.
`#Actuator kubernetes probes`

Reply:
I could but it's not really just for kubernetes. We can use that information anyway, they are not specific kubernetes endpoints.
In other words in ca be used in any infrastructure.
I've updated it to `Actuator health probes`